.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
i am wondering if this file should be specific to your own system? since not everyone is using the same editor, this won't be triggered for everyone.

Reply:
That is a good question editorconfig is not editor specific. I've seen it added somewhere else in mongo-js and I did the same, not changing much for my editor.

Reply:
ah ok gotcha